src/vs/workbench/contrib/extensions/browser/extensionEditor.ts

```diff
@@ -69,6 +69,12 @@ import { insane } from 'vs/base/common/insane/insane';
 
 function removeEmbeddedSVGs(documentContent: string): string {
 	return insane(documentContent, {
+		allowedAttributes: {
```

Comment:
Here's the list of attributes that GitHub supports: 

https://github.com/gjtorikian/html-pipeline/blob/0e3d84eb13845e0d2521ef0dc13c9c51b88e5087/lib/html/pipeline/sanitization_filter.rb#L51

I think at a minimum, `align` should be allowed on many commonly used html elements, such as `div` and `h2`. Check to see if insane lets you configure an allowed attribute for all tags. If not, then let's list off the common tags (preferably have a list of tags and use that to construct the object map instead of repeating tags in the object literal)

Reply:
Hi Matt, I added support for common attributes, as in the link you posted above